charts/opentelemetry-demo/Chart.yaml

```diff
@@ -11,9 +11,17 @@ maintainers:
   - name: tylerhelmuth
   - name: wph95
 icon: https://opentelemetry.io/img/logos/opentelemetry-logo-nav.png
-appVersion: "0.6.1-beta"
+appVersion: "1.0.0"
```

Comment:
Can we take the 1.0 images in this PR without the frontendproxy stuff?

Reply:
Yes.  The frontend proxy is an additional layer to help for browser-initiated tracing span data, which is a separate thing.  We also use it to limit the exposed ports. All that is part of the next PR.